demo/testmultivariate.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import numpy as np
+
+from sklearn.pipeline import Pipeline
+
+from sktime.classification.compose import TimeSeriesForestClassifier
+
+from sktime.transformations.panel.compose import ColumnConcatenator
+
+import matplotlib.pyplot as plt
+
+from lime import explanation
+from lime import lime_base
+import lime_timeseries as lime_ts
+
+# random dataset
+def genDataset(num_samples, num_channels, num_points):
+    X = np.random.random((num_samples, num_channels, num_points))
```

Comment:
Set initial random seed, to avoid non-deterministic test?

Reply:
commit f6686aa44e9e902f14fc6a0fe3ef184d6d49a124